Sources/URLSessionAPIAdapter.swift

```diff
@@ -8,6 +8,51 @@
 
 import Foundation
 
+// This class provides user with easy way to serialize access to a property in multiplatform environment. This class is written with future PropertyWrapper feature of swift in mind.
+internal final class Serialized<Value> {
+
+    // Synchronization queue for the property. Read or write to the property must be perforimed on this queue
+    private let queue = DispatchQueue(label: "org.funtasty.devs.ftapikit.serialization", qos: .unspecified, attributes: [], autoreleaseFrequency: .inherit, target: nil)
+
+    // The value itsef with did-set observing.
+    private var _value: Value {
```

Comment:
No other `private` property starts with underscore in this framework, I am suggesting we remove it.